src/shims-itk-wasm-dicom.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module '@itk-wasm/dicom' {
```

Comment:
why not use the packages's typescript declarations?

Reply:
`@itk-wasm/dicom` package.json's `types` property points to a missing file.

Reply:
https://github.com/InsightSoftwareConsortium/itk-wasm/pull/892